### DISCUSSION TITLE
Custom Hard/Good/Easy Timing

### DISCUSSION THREAD

**Discussion by SirAnatra** [st3v3nmw/obsidian-spaced-repetition#681] [Q&A]
There’s a way to customise how much time should pass between reviews?  Let me explain better. If I create a new card and this is hard for me to learn it will automatically be set to be learned the day after, but what if I would like to view it again in 1 hour and I wanted to see the ones I feel good about after a day? There’s a way to change the time that passes for the new cards? 

P.S. Sorry if my English was bad or unclear, I don’t speak English as my native language

**Comment by tosbourn** [comment]
From the settings panel you can adjust the numbers that feed into how things are spaced; however you can't adjust to specific timings.
